### ISSUE TITLE
"churn" (number of times a variable is changed)

### ISSUE THREAD

**Issue by BrandonKMLee** [priv-kweihmann/multimetric#26]
https://betterprogramming.pub/why-i-made-my-own-code-quality-tool-c44b40ceaafd

**Comment by priv-kweihmann** [comment]
Hi, that would be indeed an interesting metric to have - currently I'm not planning to add any new features to the tool (just the usual maintenance), but feel free to come up with a PR to make it happen.
I happily assist you if you need support

**Comment by wienans** [comment]
Actually I think it is hard to track this with a tool without a database. So calculating it via git ok file base might be good. Found this script for interested users. https://github.com/flacle/truegitcodechurn/blob/main/gitcodechurn.py

**Comment by priv-kweihmann** [comment]
> Actually I think it is hard to track this with a tool without a database. So calculating it via git ok file base might be good. Found this script for interested users. https://github.com/flacle/truegitcodechurn/blob/main/gitcodechurn.py

Hmmm, if we implement that, I would like to see it as a new optional interface, like build warnings, so we are free to choose the right tool to populate this data.
I would definitely want to avoid making git a hard dependency of the tool.
Apart from that I like the idea (using commit message history or other metatdata) as it sounds easy to implement.

**Comment by wienans** [comment]
Yes completely agree on that. And actually I think as this tool is focused on analysis of the current state of code it might be better to use 2 different tools if needed. 
I tried to execute the Skript a relatively simple git repo but couldn’t get much out of the numbers provided. 
that’s also a reason why I wouldn’t focus on this feature at the moment.
